README.md

```diff
@@ -182,7 +182,21 @@ The StarkNet network is based on the provided Ethereum endpoint. If the Ethereum
 
 ## Running with Docker
 
-The `pathfinder` node can be run in the provided Docker image. You can build the image by running:
+The `pathfinder` node can be run in the provided Docker image.
+
+```bash
+docker run \
+  -e RUST_LOG=info \
+  -p 9545:9545 \
+  eqlabs/pathfinder \
```

Comment:
Does this not require the `latest` or `v0.1.3-alpha` suffix? Or how does it pick an image version?

Reply:
It will automatically pick latest if no version is supplied. I think that's fine to keep in the README as-is for two reasons:
1. It is less maintenance work for us (otherwise we'd have to remember to update this every time we pushed a tag)
2. People running this on "serious" infra will know the best practice to use specific versioned tags

Reply:
`latest` will often be broken though as `main` isn't guaranteed to be in a working state.

Reply:
> main isn't guaranteed to be in a working state.

It's not? 👀

Reply:
Well I guess that's good because we'll know quickly if things are broken or not

Reply:
Ok - we discussed this and what we're going to do is NOT publish on merges to `main` and instead publish two images with each tag, `latest` and the tagged version